crates/sui-core/src/authority_server.rs

```diff
@@ -137,22 +135,18 @@ impl AuthorityServer {
 
 pub struct ValidatorServiceMetrics {
     pub signature_errors: IntCounter,
-    pub tx_verification_latency: Histogram,
-    pub cert_verification_latency: Histogram,
-    pub consensus_latency: Histogram,
-    pub handle_transaction_latency: Histogram,
-    pub submit_certificate_consensus_latency: Histogram,
-    pub handle_certificate_consensus_latency: Histogram,
-    pub handle_certificate_non_consensus_latency: Histogram,
+    pub tx_verification_latency: MystenHistogram,
+    pub cert_verification_latency: MystenHistogram,
```

Comment:
Not sure if tx and cert verifications are suitable with Mysten Histogram. cc @andll

Reply:
I think this is ~ok, but I do wonder if even need a histogram for those metrics (isn't just sum/avg enough)?

Reply:
landing this for now to catch the next release. Will do it in another PR if we decide to let them go